### DISCUSSION TITLE
我为什么这么倒霉啊

### DISCUSSION THREAD

**Discussion by 952E-348A** [share121/inter-knot#5072] [绝区零]
![image](https://github.com/user-attachments/assets/fb61052e-3342-4280-92c8-c74ce83c5881)
非得让我吃满保底是不是？
![image](https://github.com/user-attachments/assets/0ce0d45d-f564-4ccf-b4c0-4f1c9d29e230)
甚至这帮布池也这样

**Comment by ROBOT1145** [comment]
我也一样，凑活玩啦
